src/cli/main.cpp

```diff
@@ -39,13 +48,29 @@ int main(int argc, char* argv[]) {
 
   if (!args.empty() && (std::string_view(args[0]) == "labs")) {
     CLI::App app{"openstudio"};
+    app.name(programName);
+
+    // Preprocess the arguments, insert execute_xxx_script if a script is passed but the previous arg isn't the command.
+    // So you can ommit "execute_xxx_script" like historical behavior: `openstudio --include INCLUDE_DIR test.rb`
+    if (std::none_of(args.begin(), args.end(), [](const auto& arg) { return (arg == "execute_ruby_script") || (arg == "execute_python_script"); })) {
+      auto it = std::find_if(args.begin(), args.end(), [](const auto& arg) { return arg.ends_with(".rb") || arg.ends_with(".py"); });
+      if (it != args.end()) {
+        if (it->ends_with(".rb")) {
+          args.insert(it, "execute_ruby_script");
+        } else {
+          args.insert(it, "execute_python_script");
+        }
+      }
+    }
```

Comment:
Fix for https://github.com/NREL/OpenStudio/pull/4775#issuecomment-1460454681

We just insert the execute_xxx_script before any .rb / .py arg if missing